services/outputhost/cgcache.go

```diff
@@ -264,7 +264,7 @@ func newConsumerGroupCache(destPath string, cgDesc shared.ConsumerGroupDescripti
 		cfgMgr:                   h.cfgMgr,
 	}
 
-	cgCache.checkSingleCGVisibleCache = cache.NewLRUWithInitialCapacity(1, 1 << 6)
+	cgCache.checkSingleCGVisibleCache = cache.NewLRUWithInitialCapacity(1, 1<<6)
```

Comment:
as we discussed in person, looks like the LRU cache is an overkill here and can be avoided.. we can address it as a separate patch..